src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -793,6 +793,22 @@ private IntermediateNode InjectCoercion(IntermediateNode child, IRTranslatorCont
                     case CoercionKind.AggregateToDataEntity:
                         unaryOpKind = UnaryOpKind.AggregateToDataEntity;
                         break;
+                    case CoercionKind.UntypedToText:
```

Comment:
>UntypedToText [](http://example.com/codeflow?start=38&length=13)

I think it's good that these directly map to the corresponding function calls - easier to explain the semantics.